operator/src/lib.rs

```diff
@@ -120,7 +119,7 @@ impl PodInformation {
     }
 }
 
-/// Count the pods stored in each (hashed) selector
+/// Retrieve all pods encapsulated for a spark node in selector/hash -> Vec<Pod>
```

Comment:
I'm not sure if `encapsulated` is the right word here. I'm not sure what you mean.